source/agora/app.d

```diff
@@ -72,6 +72,8 @@ private int main (string[] args)
         auto router = new URLRouter();
 
         auto node = new Node!NetworkManager(config);
+        scope (exit) node.shutdown();
```

Comment:
```suggestion
        scope(exit) node.shutdown();
```

Reply:
Ah, really? Ok, I used it like this before because it's not technically a function call. :)